tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -2540,7 +2540,7 @@ def max_phrase(x,y):
 def test_max_property(language):
     @types('int[:]')
     def max_call(x):
-        return x.max()
+        return x.amax()
```

Comment:
This code does not run in python. The ndarray method is called `max` (see [doc](https://numpy.org/doc/stable/reference/generated/numpy.ndarray.max.html) )